src/main.cpp

```diff
@@ -19,8 +19,8 @@ int main(int argc,char *argv[])
 			hide = true;
 	}
 
-	App		myApp( argc>1 ?  argv[argc-1] : "");	
+	App		myApp(  argv[argc-1]);	
```

Comment:
You can change the function signature to be `const char* argv[]` instead of `char* argv[]` and then revert this change.